packages/core/chrome/core-chrome-browser-internal/src/project_navigation/project_navigation_service.ts

```diff
@@ -169,6 +188,68 @@ export class ProjectNavigationService {
     };
   }
 
+  private initNavigation(
+    navTreeDefinition: Observable<NavigationTreeDefinition>,
+    { navLinksService, cloudUrls }: { navLinksService: ChromeNavLinks; cloudUrls: CloudURLs }
+  ) {
+    if (this.navigationTree$.getValue() !== undefined) {
+      throw new Error('Project navigation has already been initiated.');
+    }
+
+    const deepLinksMap$ = navLinksService.getNavLinks$().pipe(
+      map((navLinks) => {
+        return navLinks.reduce((acc, navLink) => {
+          acc[navLink.id] = navLink;
+          return acc;
+        }, {} as Record<string, ChromeNavLink>);
+      })
+    );
+
+    const cloudLinks = getCloudLinks(cloudUrls);
+
+    this.navigationTree$
+      .pipe(
+        takeUntil(this.stop$),
+        filter((navTree): navTree is ChromeProjectNavigationNode[] => Boolean(navTree))
+      )
+      .subscribe((navTree) => {
+        this.projectNavigationNavTreeFlattened = flattenNav(navTree);
+        this.setActiveProjectNavigationNodes();
+      });
+
+    combineLatest([navTreeDefinition.pipe(takeUntil(this.stop$)), deepLinksMap$])
+      .pipe(
+        map(([def, deepLinksMap]) => {
+          return parseNavigationTree(def, {
+            deepLinks: deepLinksMap,
+            cloudLinks,
+          });
+        }),
+        catchError((e) => {
```

Comment:
Redundant?

Reply:
Yep.. Fixed in https://github.com/elastic/kibana/pull/174544/commits/ea7cb38a6e37ccac827ded451ca62ebeea83bfd1